dask/dataframe/io/parquet/arrow.py

```diff
@@ -63,7 +63,15 @@ def _append_row_groups(metadata, md):
 
 
 def _write_partitioned(
-    table, root_path, filename, partition_cols, fs, index_cols=(), **kwargs
+    table,
+    df,
+    root_path,
+    filename,
+    partition_cols,
+    fs,
+    preserve_index,
```

Comment:
presumably we don't need to worry about the order changing since this is a private function?

Reply:
Right - I assumed explcit arguments were probably better since the funciton is private. However, I have no problem using kwargs if you think this is an unncessary risk